### PR TITLE
VATRP-3012: Statusbar and Call-Info

### PR DESCRIPTION
**Pull request by Semrubo**

### CODE REVIEW

**Comment by shareefalis** [comment]
+1

**Comment by zackm0571** [comment]
Please fix conflicts

**Comment by shareefalis** [comment]
https://github.com/VTCSecureLLC/ace-ios/pull/358

**Comment by shareefalis** [comment]
If it builds, merge it.

**Comment by zackm0571** [comment]
Do not merge, archive failed

File: submodules/belle-sip [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit e57cf673373fc611852289691c46fd56e78ae041
+Subproject commit 03541f3a802290133c20dd84a2b8cac1d9d02ffd
```
Comment (ianblenke) [inline submodules/belle-sip:1]: Why are we changing this submodule?

File: submodules/cmake-builder [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 96b15de85777af96327cff9e02fb5d46ed3007fd
+Subproject commit 2bb01e4fb51d22fac0118dfbcb9bc64933a7c592
```
Comment (ianblenke) [inline submodules/cmake-builder:1]: Why are we changing this submodule?

File: submodules/linphone [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit 82e395bc9b140f7f97b7ab9b41d8e5aa41f70965
+Subproject commit 0e996357d70a606a86d97af7d28eaa8451f09952
```
Comment (ianblenke) [inline submodules/linphone:1]: Why are we changing this submodule?

**Comment by kennethdick** [comment]
Revert submodule changes!!!

**Comment by zackm0571** [comment]
Closed for #367

**Pull request closed without merging**